remarkablemark/html-react-parser
# Replace HTML to DOM implementation with `html-dom-parser` package

#### Summary:
Updating the package and tests now that the HTML to DOM logic has been externalized to [html-dom-parser](https://www.npmjs.com/package/html-dom-parser) package.

#### Tasks:
- Replace HTML to DOM implement with `html-dom-parser`
  - Save to dependencies:
    - `html-dom-parser`@0.0.2
  - Remove from dependencies:
    - `domhandler`@2.3.0
    - `htmlparser2`@3.9.1
  - Remove `lib/html-to-dom-server.js` and `lib/html-to-dom-client.js` and require `html-dom-parser`
- Tests
  - Replace with `html-dom-parser` where necessary
  - Prune and remove unused tests and helpers
  - Tidy tests, mocks, and helpers
  - Add test for doctype (should be ignored by parser)
- README
  - Update **Special Thanks** section
  - Tidy where necessary
